test/utmost/setup.js

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const Utmost = artifacts.require('UtmostTest');
+const BN = require('bn.js');
+const web3 = require('./../test_lib/web3');
+
+const { AccountProvider } = require('../test_lib/utils.js');
+
+contract('Utmost::setup', (accounts) => {
+  let utmost;
+  let consensusCogateway;
+  let initialSupply;
+  let coconsensus;
+  const accountProvider = new AccountProvider(accounts);
+
+  beforeEach(async () => {
+    coconsensus = accountProvider.get();
+    initialSupply = new BN('1000000');
+    utmost = await Utmost.new(coconsensus, initialSupply);
+    consensusCogateway = '0x0000000000000000000000000000000000004d02';
+  });
+
+  it('should setup Utmost token correctly.', async () => {
+    await utmost.setup({ from: coconsensus });
+
+    assert.strictEqual(
+      web3.utils.isAddress(utmost.address),
+      true,
+      'Utmost token contract address must not be zero.',
```

Comment:
This check is not needed. The contract is deployed in before each section and already setup function is called on this contract.

Reply:
Done!